src/coreclr/debug/di/rspriv.h

```diff
@@ -1200,7 +1200,6 @@ class CordbCommonBase : public IUnknown
         {
             STRESS_LOG1(LF_ASSERT, LL_ALWAYS, "CordbCommonBase::IsValidObject() failed: %x\n", this);
             _ASSERTE(!"CordbCommonBase::IsValidObject() failed");
-            FreeBuildDebugBreak();
         }
```

Comment:
Delete the whole `ValidateObject` method? I see it used only on one place that has link to some ancient DevDiv bug.

Reply:
Sure.